package.json

```diff
@@ -33,7 +33,7 @@
     "tslint": "^5.12.0",
     "tslint-config-prettier": "^1.17.0",
     "tslint-config-semistandard": "^7.0.0",
-    "typescript": "^3.2.2"
+    "typescript": "^3.7.5"
```

Comment:
Bonus 🔥

Reply:
Ha yeah, I noticed a weird type error and this fixed it.